_artwork/remnant.md

```diff
@@ -1,5 +1,5 @@
 ---
 full: /images/artwork/remnant.jpg
 thumb: /images/artwork/remnant_thumb.jpg
-alt: Wanderer ships
+alt: Remnant ships
```

Comment:
Should it just say "Remnant" to fit with the others?

Reply:
Good point